AEPLifecycle/Sources/Lifecycle.swift

```diff
@@ -32,7 +32,7 @@ public class Lifecycle: NSObject, Extension {
     public required init(runtime: ExtensionRuntime) {
         self.runtime = runtime
         lifecycleState = LifecycleState(dataStore: NamedCollectionDataStore(name: name))
-        lifecycleV2 = LifecycleV2(dataStore: NamedCollectionDataStore(name: name))
+        lifecycleV2 = LifecycleV2(dataStore: NamedCollectionDataStore(name: name), dispatch: MobileCore.dispatch(event:))
```

Comment:
Can we add small comment before `lifecycleState` and `lifecycleV2` declaration on what they handle?